pkg/ingester/ingester_ring_test.go

```diff
@@ -85,5 +89,67 @@ func TestRingConfig_CustomConfigToLifecyclerConfig(t *testing.T) {
 	expected.Addr = cfg.InstanceAddr
 	expected.ListenPort = cfg.ListenPort
 
-	assert.Equal(t, expected, cfg.ToLifecyclerConfig())
+	logger := log.NewNopLogger()
+	tokenGenerator, err := ring.NewSpreadMinimizingTokenGenerator(expected.ID, expected.Zone, cfg.SpreadMinimizingZones, logger)
+	assert.NoError(t, err)
```

Comment:
It's better to halt the test on error.
```suggestion
	require.NoError(t, err)
```